components/Logo/logo-ant.svg

```diff
@@ -0,0 +1 @@
26 3.007-.126h.122c1.077.004 2.088.052 2.97.134l-3.051-8.12zm-10.17-14.056s1.517 4.752 1.517 5.594c0 .58-.335 1.069-.812 1.326.393.127.738.35 1.008.65.603-.524 1.49-1.011 2.765-1.011 1.458 0 2.71.636 3.816 1.198.352.178.693.35 1.01.483.235-.25.564-.409.932-.409.714 0 1.293.581 1.293 1.298 0 .717-.579 1.298-1.293 1.298-.58 0-1.052-.39-1.215-.917-.424-.162-.858-.38-1.304-.607-.978-.496-2.087-1.059-3.239-1.059-1.107 0-1.797.515-2.192.951.016.11.034.22.034.334a2.325 2.325 0 01-2.32 2.33 2.325 2.325 0 01-2.32-2.33c0-.114.018-.225.033-.335-.395-.437-1.086-.95-2.191-.95-1.152 0-2.261.564-3.24 1.06-.446.227-.88.445-1.303.607-.164.528-.636.917-1.215.917a1.296 1.296 0 01-1.293-1.299c0-.717.579-1.298 1.293-1.298.368 0 .697.16.932.409.316-.133.657-.303 1.009-.483 1.105-.562 2.36-1.198 3.817-1.198 1.274 0 2.162.487 2.765 1.011a2.3 2.3 0 011.008-.65 1.507 1.507 0 01-.813-1.326c0-.842 1.518-5.594 1.518-5.594zM38.563 16.386s-1.697.145-2.82.751c0 0 4.47-.024 6.55 1.72 0 0-5.06-1.551-10.956 2.302 0 0 9.378-1.794 13.824 2.01 0 0-1.435-2.65-4.452-3.22 0 0 3.433-.136 4.609.434 0 0-1.016-2.183-2.167-2.704v-1.556c0-.316-1.719-1.918-4.588-2.189v2.452z" fill="#fff"/><path d="M37.8 23.347s.723.63 1.029 1.211c0 0-.662.653-1.294.974 0 0 1.915.602 2.42 1.314-1.021-.32-2.765-.2-3.185.665 1.166-.126 1.88.175 2.03.62 0 0-.278.4-1.126.576-.904.189-1.134.564-1.134 1.087 0 .524.686 1.003 1.932.964 1.244-.04 4.639-.552 4.895-6.962-.256-.603-3.357-1.833-5.576-1.82l.008 1.37zM43.788 30.439c1.523-2.367 2.357-4.43 2.5-6.187 2.257 2.502 2.12 5.431 1.738 7.673a16.581 16.581 0 00-1-2.226c.27 3.457-.756 5.328-3.076 5.614a18.248 18.248 0 01-.162-4.874zM17.257 10.672c-.424.65-1.044 2.457-1.862 5.422-1.952 2.23-3.86 5.343-5.722 9.337A19.49 19.49 0 006.8 26.693c2.65-4.988 5.239-8.719 7.767-11.191a12.618 12.618 0 012.69-4.83zM22.29 15.782l-6.177 2.91c1.76.029 3.158-.29 4.193-.959 1.036-.668 1.697-1.319 1.984-1.95zM20.01 14.722c-.034-1.271-.2-1.986-.497-2.144-.33-.147-.521-.463-.33-1.042.119-.367.083-.776-.107-1.227l1.676-1.118 1.886 1.187c.066.419-.071.805-.411 1.158.41.444.41 1.017-.065 1.68.013.163.022.324.027.482.008.238-.001.58-.027 1.024H20.01z" fill="#fff" fill-opacity=".6"/><path d="M42.057 32.887c-.265 4.878.764 7.65 3.086 8.316 3.483.999 9.676 1.132 9.28 8.698.314 1.002.602 1.953.863 2.856a27.75 27.75 0 01-3.312 3.013c-1.158-2.826-2.917-4.603-4.952-5.87-8.02-5.043-9.675-10.715-4.965-17.013zM19.503 49.714c4.68 11.73 12.174 4.214 17.962 6.962 2.647 1.256 4.091 3.563 4.332 6.919a31.002 31.002 0 01-7.024.801 30.828 30.828 0 01-13.107-2.904c-.014-3.532-.736-7.457-2.163-11.778zM28.383 17.997c1.214-.335 2.207-1.091 2.977-2.268-.049.7-.225 1.304-.528 1.81 3.51-.63 6.233-2.753 8.171-6.368-3.573-1.94-7.256-.616-9.23 1.796-1.317 1.609-1.78 3.285-1.39 5.03z" fill="#fff" fill-opacity=".6"/><defs><linearGradient id="paint0_linear" x1="124" y1="-22" x2="23.527" y2="-26.765" gradientUnits="userSpaceOnUse"><stop stop-color="#02E9F8"/><stop offset="1" stop-color="#01C2FF"/></linearGradient></defs></svg>
```

Comment:
Maybe we can name these differently, like `logo-blue` / etc.

But I also wonder: shouldn't we just use the same logo with transparency so that it works with any background?